tests/test_protocol.py

```diff
@@ -200,10 +200,31 @@ def stream_reader(request):
     protocol.pause_writing()
     protocol._stream_writer.write(b"1234")
 
-    with pytest.raises(SMTPTimeoutError):
+    with pytest.raises(SMTPTimeoutError) as exc:
         await protocol._drain_writer(timeout=0.01)
 
     protocol._stream_writer.close()
+    assert str(exc.value) == "Timeout on send request"
+
+
+async def test_protocol_timeout_readline(
```

Comment:
Ideally I'd also like to see this in` test_timeouts.py` but since some of the fixtures are local to this file, it's probably better to leave it here for now.

Reply:
Move all test into test_timeouts